src/Controls/src/Core/HandlerImpl/Window/Window.Impl.cs

```diff
@@ -66,6 +66,9 @@ public Window()
 		public Window(Page page)
 			: this()
 		{
+			if (this is Element windowElement)
```

Comment:
Do we need this if statement?

Reply:
This seems strange to me to since it's the ctor of a concrete implementation the Parent property should be available already, you shouldn't need to have `if(this is Element windowElement)`